Makefile.pre.in

```diff
@@ -1450,7 +1451,8 @@ LIBSUBDIRS=	tkinter tkinter/test tkinter/test/test_tkinter \
 		multiprocessing multiprocessing/dummy \
 		unittest unittest/test unittest/test/testmock \
 		venv venv/scripts venv/scripts/common venv/scripts/posix \
-		curses pydoc_data
+		curses pydoc_data \
+		zoneinfo
```

Comment:
nitpick:

```suggestion
		curses pydoc_data zoneinfo		
```

Reply:
What's the rule here? I don't know what pydoc_data is, but it seems like everything else is separated along conceptual lines (unittest stuff by itself, venv stuff by itself, asyncio by itself, etc).

Reply:
There is no rule. Only personal perferences. I would expect sorting, but it seems like we lost that long time ago.

Reply:
You're free to ignore my coding-style comment ;-)

Reply:
OK, I think I prefer it to be separated conceptually. Would be good to actually have a rule at some point (maybe with an autoformatter). I'm going to go ahead and merge.